lib/js/components/TabItem/TabItem.vue

```diff
@@ -1,67 +1,122 @@
 <template>
-	<div class="a-tabItem" :title="title" :class="{ '-isActive': isActive }">
-		<wnl-icon :icon="icon" :size="ICON_SIZES.X_SMALL" />
+	<div
+		class="tabItem"
+		:title="label"
+		:class="{
+			'-sizeMedium': size === TAB_ITEM_SIZES.MEDIUM,
+			'-sizeSmall': size === TAB_ITEM_SIZES.SMALL,
+			'-isSelected': isSelected,
+		}"
+		@click="$emit('click')"
+	>
+		<ds-icon v-if="icon !== null" class="tabItem__icon" :icon="icon" :size="iconSize" />
+		<span class="tabItem__label">{{ label }}</span>
```

Comment:
In figma, I just found that `label` is optional. We sould add `v-if` here and make prop optional